src/containers/pages/JurisdictionAssignment/JurisdictionTable/index.tsx

```diff
@@ -116,11 +132,17 @@ const JurisdictionTable = (props: JurisdictionSelectorTableProps) => {
     selectedLeafNodes,
     plan,
     jurisdictionsMetadata,
+    mapCurrentParent,
+    fetchUpdatedCurrentParentActionCreator,
     serviceClass,
     autoSelectionFlow,
     deselectAllNodesCreator,
   } = props;
 
+  function fetchUpdatedCurrentParentHandler(currentParentId: string, isRootJurisdiction: boolean) {
+    fetchUpdatedCurrentParentActionCreator(currentParentId, isRootJurisdiction);
+  }
```

Comment:
Is it necessary to wrap `fetchUpdatedCurrentParentActionCreator` like this?

Reply:
This was to avoid having to transform the `HeaderBreadCrumb` component to connect it to store and just have a handler passed to it as a prop to be able to dispatch the `fetchUpdatedCurrentParent` action.

Reply:
Why do we need to dispatch that action creator when you click on the link?  I would like us to find a way to avoid that.